docs/src/ambiguities.md

```diff
@@ -0,0 +1,21 @@
+# Ambiguities
+
+Method ambiguities are cases where multiple methods are applicable to a given set of arguments, without having a most specific method.
+
```

Comment:
```suggestion

## Examples
```
Adding example section would be better for consistency with `unbound_args.md` and `piracies.md` pages.